torch/_inductor/codecache.py

```diff
@@ -779,36 +857,38 @@ class CompiledFxGraph:
     to support FxGraph caching.
     """
 
-    compiled_artifact: Optional[Callable[..., Any]] = None
-    current_callable: Optional[Callable[..., Any]] = None
-    cache_key: Optional[str] = None
-    artifact_path: Optional[str] = None
-    cache_linemap: Optional[List[Tuple[int, str]]] = None
-    device_types: Set[str] = field(default_factory=set)
-    device_idxs: Set[int] = field(default_factory=set)
-    mutated_inputs: Set[str] = field(default_factory=set)
-    mutated_input_idxs: Set[int] = field(default_factory=set)
-    constants: Dict[str, torch.Tensor] = field(default_factory=dict)
-    output_strides: Optional[List[Optional[Tuple[int, ...]]]] = None
+    compiled_artifact: Optional[Callable[..., Any]]
```

Comment:
These are all required fields set in the init now, so no need for the default values (and less confusing)